cli/tests/extensions/mod.rs

```diff
@@ -7,6 +7,7 @@ use std::sync::Mutex;
 
 use lazy_static::lazy_static;
 use phylum_cli::commands::extensions::extension::Extension;
+use phylum_cli::commands::extensions::permissions::{Permission, Permissions};
```

Comment:
This warns on Windows.

```suggestion
#[cfg(unix)]
use phylum_cli::commands::extensions::permissions::{Permission, Permissions};
```